docs/source/integrations/cvat.rst

```diff
@@ -0,0 +1,1034 @@
+    auth = {
+        "username": ...,
+        "password": ...,
+    }
+
+    info = view.annotate(label_field="ground_truth", auth=auth)
+
+4. FiftyOne annotation config
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can store any CVAT configuration setting or credentials in your FiftyOne
+annotation config located at `~/.fiftyone/annotation_config.json`:
+
+.. code-block:: text
+
+    {
+        "cvat_username": ...,
+        "cvat_password": ...,
+        "cvat_url": "localhost",
+        "cvat_port": 8080,
+        "cvat_https": false
+    }
+
+.. warning:
+
+    Storing your username and password in plain text on disk is generally not
+    recommended. Consider using environment variables instead.
+
+.. _cvat-annotation:
+
+Annotation
+__________
+
+Use the
+:meth:`annotate() <fiftyone.core.collections.SampleCollection.annotate>` method
+to send samples and optionally existing labels to CVAT for annotation.
+
+The :meth:`annotate() <fiftyone.core.collections.SampleCollection.annotate>`
+method provides various keyword arguments that you can use to customize the
+annotation tasks that you wish to be performed.
+
+**General arguments**
+
+- `backend`: the annotation backend to use. Use `"cvat"` for the CVAT backend
+- `media_field`: (`"filepath"`) the sample field containing the path to the
+  source media to upload
+- `launch_editor`: whether to launch the annotation backend's editor after
+  uploading the samples
+
+**Label schema**
+
+- `label_schema`: the complete dictionary description of the annotation schema
+  to use
+- `label_field`: the name of a single label field to upload or create
+- `label_type`: if `label_field` is used to create a new field, this specifies
+  the type of field to create. Supported values are (`classification`,
+  `classifications`, `detections`, `keypoints`, `polylines`, `scalar`). If
+  provided, this will be the default type for any label fields in
+  `label_schema` whose types are not otherwise specified
+- `classes`: a list of classes to upload or create when `label_field` is given.
+  If provided, this will be used to define the classes list for any label
+  fields in `label_schema` that do not otherwise have class lists specified
+- `attributes`: a list of label attributes to upload or create when
+  `label_field` is given, or a dict mapping attribute names to the type of
+  annotation widget to use (e.g., `text`, `select`, etc). If provided, this
+  will define the default attributes for any label fields in `label_schema`
+  that do not otherwise have their attributes specified
+
+**CVAT-specific arguments**
+
+- `segment_size`: the maximum number of images to upload per job. Not
+  applicable to videos
+- `image_quality`: an int in `[0, 100]` determining the image quality to upload
+  to CVAT
+- `task_assignee`: a username to assign the generated tasks
+- `job_assignees`: a list of usernames to assign jobs
+- `job_reviewers`: a list of usernames to assign job reviews
+
+.. _cvat-label-schema:
+
+Label schema
```

Comment:
I combined the `Label schema` and `Attributes` sections and spent some time trying to make it a bit easier to understand all the different options at one's fingertips. Hopefully you agree it reads a bit easier now!

A latent goal while doing my editing was to "anonymize" as much of the copy as possible by removing unnecessary mentions of "CVAT" so that much of this can be upgraded to an `Annotations` user guide page soon.